tools/topology/topology2/include/common/tokens.conf

```diff
@@ -52,6 +52,7 @@ Object.Base.VendorToken {
 		curve_type		260
 		curve_duration		261
 		init_value		262
+		use_topology_name	263
```

Comment:
Which other modules do we need to add this attribute too ?

Reply:
Are there any other modules creating alsa mixers? The PCM names can be independently named already so I do not see a need for anything else.

Reply:
"use_topology_name" seems to mean "do_not_add_prefix_use_topology_name_as_is" 

It's not a replacement of a name by a topology-defined one, more an enforcement to use as is.

Did I get this right? if yes, the naming is questionable.

Reply:
Yes, you got it. I try to come up with a better name.